src/formatDate.js

```diff
@@ -50,7 +50,51 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateRes = date.split(fromFormat[3]);
+  const result = [];
+  let year = 0;
```

Comment:
You can add space between const and let variables it will improve readability and code style